include/supercluster.hpp

```diff
@@ -158,6 +190,62 @@ class Supercluster {
         return result;
     }
 
+    GeoJSONFeatures getChildren(std::uint32_t cluster_id) {
+        const auto origin_id = cluster_id >> 5;
+        const auto origin_zoom = cluster_id % 32;
+
+        const auto zoom_iter = zooms.find(origin_zoom);
+        if (zoom_iter == zooms.end())
+            throw std::runtime_error("No cluster with the specified id.");
+
+        auto &zoom = zoom_iter->second;
+        if (origin_id >= zoom.clusters.size())
+            throw std::runtime_error("No cluster with the specified id.");
+
+        const double r = options.radius / (double(options.extent) * std::pow(2, origin_zoom - 1));
+        const auto &origin = zoom.clusters[origin_id];
+
+        GeoJSONFeatures children;
+
+        zoom.tree.within(origin.pos.x, origin.pos.y, r, [&, this](const auto &id) {
+            const auto &c = zoom.clusters[id];
```

Comment:
Do we need to protect account clusters that can't be found here?

Reply:
No — this should be guaranteed by KDBush `within`.

Reply:
cool; can we add an `assert` before the `operator[]`?